build/build.gradle

```diff
@@ -131,7 +131,12 @@ ext {
 		'eu.esdihumboldt.hale.common.cache.test',
 		'eu.esdihumboldt.util.blueprints.entities.test',
 		'eu.esdihumboldt.hale.io.oml',
-		'eu.esdihumboldt.hale.io.oml.test'
+		'eu.esdihumboldt.hale.io.oml.test',
+		// exclude them because org.deegree needs to be updated with jaxb dependencies
```

Comment:
Do they need to be excluded if the feature is removed from the HALE product and the test bundle from the tests product?

Reply:
Not sure. I'll try to remove and see.